packages/office-ui-fabric-react/src/common/_semanticSlots.scss

```diff
@@ -55,3 +55,6 @@ $listHeaderBackgroundPressedColor: "[theme:listHeaderBackgroundPressed, default:
 $listItemBackgroundHoveredColor: "[theme:listItemBackgroundHovered, default: #f8f8f8]";
 $listItemBackgroundCheckedColor: "[theme:listItemBackgroundChecked, default: #eaeaea]";
 $listItemBackgroundCheckedHoveredColor: "[theme:listItemBackgroundCheckedHovered, default: #d0d0d0]";
+
+/* Keytips */
+$keytipDisabledTextColor: "[theme:keytipDisabledText, default: #b1b1b1]";
```

Comment:
keytips should not get their own semantic slot
semantic slots are not for each feature to have their own, but for each concept
can you reuse an existing concept?

Reply:
How can I reuse an existing one if the color isn't shared with anyone else? I thought your previous comments in the other PRs (the part 1 and 2 ones) said to follow this pattern

Reply:
All colors should be one that's in our palette, we shouldn't be adding things with bespoke colors.
For now, just use one of the palette slots from here:
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/common/_themeOverrides.scss

Reply:
Now using neutralTertiaryAlt instead which is very close to our original color